kong/tools/queue_schema.lua

```diff
@@ -39,12 +39,14 @@ return Schema.define {
       initial_retry_delay = {
         type = "number",
         default = 0.01,
+        between = { 0.001, 1000000 }, -- effectively unlimited maximum
```

Comment:
Just being curious. Is there a reason why we have this "1000000" limit instead of just "gt = 0.001"?

Reply:
> Just being curious. Is there a reason why we have this "1000000" limit instead of just "gt = 0.001"?

No, thank you for pointing that out.  I was not aware of `gt`, but what we'd really need is `ge`.  I'll leave it at this for now.